Detectors/MUON/MID/Clustering/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+set(MODULE_NAME "MIDClustering")
+
+O2_SETUP(NAME ${MODULE_NAME})
+
+set(SRCS
+   src/Clusterizer.cxx
+   src/ClusterizerDevice.cxx
+)
+
+set(LIBRARY_NAME ${MODULE_NAME})
+set(BUCKET_NAME mid_clustering_bucket)
+
+O2_GENERATE_LIBRARY()
+
+O2_GENERATE_EXECUTABLE(
+  EXE_NAME runMIDClusterizer
+  SOURCES src/runMIDClusterizer.cxx
+  MODULE_LIBRARY_NAME ${LIBRARY_NAME}
+  BUCKET_NAME ${BUCKET_NAME}
+)
+
+install(FILES config/runMIDclusterizer.json
```

Comment:
Hi @dstocco I believe what you want to write is:

```cmake
install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/config/runMIDclusterizer.json DESTINATION bin/config)
```

In practice you need to specify explicitly what is the source path. The way the sandbox works has absolutely nothing to do with it, and if it works on your computer it does that by chance :-)

As for the installation part, I don't like a configuration file to end under `bin` (or a subdir of it), this is wrong. It should go to `<installation_prefix>/etc`.

Reply:
Ciao @dberzano,
thanks for the hint. I added the ${CMAKE_CURRENT_SOURCE_DIR} and installed in etc/config instead of bin. However, as you can see, it still fails...

Cheers,
Diego